### ISSUE TITLE
DMP 0.25.0 label values with blank do not work anymore

### ISSUE THREAD

**Issue by derkoe** [fabric8io/docker-maven-plugin#988]
### Description

This works with 0.24.0 but fails with 0.25.0

      <plugin>
        <groupId>io.fabric8</groupId>
        <artifactId>docker-maven-plugin</artifactId>
        <version>${docker-maven-plugin.version}</version>
        <configuration>
          <images>
            <image>
              <name>my-super-image:%l</name>
              <build>
                <from>my-super-baseimage</from>
                <assembly>
                  <descriptorRef>artifact</descriptorRef>
                </assembly>
                <labels>
                  <vendor>My Company</vendor>
                </labels>
              </build>
            </image>
          </images>
        </configuration>
      </plugin>

Error message:

     [ERROR] Failed to execute goal io.fabric8:docker-maven-plugin:0.25.0:build (default-cli) on project myproject: Unable to build image [my-super-image:latest] : Syntax error - can't find = in "Company". Must be of the form: name=value (Internal Server Error: 500) -> [Help 1]

### Info

* d-m-p version : 0.25.0
* Maven version (`mvn -v`) : 
```
Apache Maven 3.5.2 (138edd61fd100ec658bfa2d307c43b76940a5d7d; 2017-10-18T09:58:13+02:00)
Java version: 1.8.0_162, vendor: Oracle Corporation
Default locale: de_AT, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```
* Docker version : 18.03.0-ce
* If it's a bug, how to reproduce : add the config above to your pom.xml, run mvn docker:build

**Comment by rhuss** [comment]
Seem like I had added that regression. My bad, going to fix this asap, you can expect a 0.25.1 very soon.

**Comment by derkoe** [comment]
Thanks for the really fast fix
